dspace-api/src/main/java/org/dspace/scripts/Process.java

```diff
@@ -211,6 +221,14 @@ public Date getCreationTime() {
         return creationTime;
     }
 
+    public List<Group> getGroups() {
```

Comment:
We should make it clear that these only return *Special Groups* associated with the Process and not all Groups.... that could be done by adding JavaDocs here, or by adding a comment where `process2group` is referenced above.   It's just important for developers to understand what these groups are when looking at the Process object.